N/norMD/build_tarballs.jl

```diff
@@ -0,0 +1,64 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "norMD"
```

Comment:
I would suggest to use a less concise name as the tool is for a specific field (bioinformatics, MSA), e.g.,
```suggestion
name = "AquaMSANorMD"
```
Even adding MSA to disambiguate among all the "Aqua"'s out there (Aqua.jl, Aqua macOS GUI, JetBrains Aqua, ... https://github.com/search?q=aqua&type=repositories&s=stars&o=desc ).

Reply:
Hi! Thanks, I understand :) Can it be `NormalizedMeanDistance` ?

Reply:
From my perspective, with a background in Computer Science, and not in Bioinformatics, I would be more comfortable with a name completely without field-specific abbreviations, i.e., I would really be most comfortable with a name like
```
AquaMultipleSequenceAlignmentNormalizedMeanDistance
```
But I'm OK with AquaMSA as the prefix if that is preferred, but I think there should be a prefix to disambiguate among other interpretations of "NorMD" or "NormalizedMeanDistance".

I do not think NormalizedMeanDistance only makes sense for MSA (with sequence referring to DNA/RNA sequences, I presume...). E.g., in Computer Vision, it is common to look at the IoU (intersection over union) distance between object detections, and then compute `mAP` (mean average precision) based on an IoU threshold. I would not expect everyone in the whole world (Julia world) to understand, or even accept, computer vision-specific terms as universal terms.

Reply:
In the last commit, I changed the name from norMD to AQUAnorMD to better align with the upstream project and improve clarity.
Should the file also be renamed and moved into the A/ folder to reflect this new naming?

Reply:
> I think that the main concern with naming Yggdrasil recipes should be that of the Julia General registry

That's not a concern, JLL packages don't necessarily need to follow regular packages naming.

Reply:
Not necessarily, I guess so, but perhaps in e.g. niche cases like this it's a nice default?

See also https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/1379

Reply:
Thanks for the feedback and the thoughtful discussion.

I'd like to explain why I still think `norMD` is the most appropriate name for the JLL package:

1. **There is no other standalone software named `norMD`**. So, currently there are no naming clashes, and such conflicts are unlikely in the future — especially since the casing (`norMD`) isn’t standard in most Julia packages.

2. **I really like the idea of JLL packages being named after their actual software or suite**. It helps a lot with discoverability. For example, that’s how I found useful tools like `MAFFT`. I wouldn’t have found them easily if the name had been something like `MultipleAlignmentUsingFastFourierTransform_jll`.

3. **While I’ve taken the implementation from AQUA because it’s the only publicly accessible one — available via `wget` — the `norMD` tool was published independently before AQUA**. In fact, to get other versions, you'd need to contact the paper authors directly. So AQUA is just bundling `norMD`, not the origin of it. Future versions of the tool could appear outside AQUA too. So, I would prefer not to tie the package name to AQUA.

Let me know what you think — I’m open to discussing further, but I believe keeping the `norMD` name fits the conventions and benefits users the most.

Reply:
Note that the mentioned `normD` and `normd` do not clash because of the different casing ;) Also, `normD` is an R subroutine, so it would not be a JLL package.

Reply:
I would say that casing is too brittle to rely on for disambiguation.

And yes, I don't see the R package command becoming a JLL, it was more to show that there were (more common?) interpretations of what "normd" might be.